_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+Legal aspects of renting in Switzerland
+=======================================
+
+This page provides a brief overview of some legal aspects of apartment rental in Switzerland, in the hopes that it will help you navigate them in the future.
+
+## Lease takeovers
+In many countries, one can cancel one’s lease at any date as long as a certain notice period is respected. In Switzerland, this is usually not the case. A typical lease is valid for one year, and tacitly renewed. When you want to leave, you can only do so at its next term, and you also need to inform the landlord 3 months in advance of your intention of moving out. So, if you have moved into an apartment on 21.06.2020, and you want to move out in 2023, you will have to pay rent until 21.06.2023, and your notice should be received by your landlord no later than 21.03.2023 (https://bail.ch/bail/page/faq/10).
```

Comment:
Nice that you have inline references. I personally find it quite useful when reading such articles.

When I merge this, I'm planning to rewrite all reference links in the following format:

no later than 21.03.2023 ([bail.ch](https://bail.ch/bail/page/faq/10)).

`no later than 21.03.2023 ([bail.ch](https://bail.ch/bail/page/faq/10)).`

That way, the URL doesn't distract from the text and the link indicates where it goes (unlike e.g. linking the word "here"). I think I'll keep footnotes only for tangential remarks and not for sources. Is that okay with you?